style.css

```diff
@@ -0,0 +1,17 @@
+.button.play {
+    box-sizing: border-box;
+    width: 74px;
+    height: 74px;
+    border-style: solid;
+    border-width: 37px 0px 37px 74px;
```

Comment:
maybe resize?

Reply:
we will work it out once we get everything working :)